query-node/mappings/src/common.ts

```diff
@@ -188,3 +208,21 @@ class Logger {
 }
 
 export const logger = new Logger()
+
+/*
+  Helper for converting Bytes type to string
+*/
+export function convertBytesToString(b: Bytes | null): string {
+  if (!b) {
+    return ''
+  }
+
+  const result = Buffer.from(b.toU8a(true)).toString()
+
+  // prevent utf-8 null character
+  if (result.match(/^\0$/)) {
```

Comment:
I don't this will work for multiple null characters, for example: `0x000000`

![image](https://user-images.githubusercontent.com/19978260/126769123-aad15634-e7da-4766-86f1-3aa7c80e09c2.png)

Reply:
I feel like we fixed this issue couple of times before, but here we are again... Thx for spotting. Fixed